style.css

```diff
@@ -0,0 +1,82 @@
+body {
+    margin: 0;
+    padding: 0;
+    background-color: var(--background-main-color);
+
+}
+
+:root {
+    --border-nav-color: #FFEAD5;
+    --background-nav-color: white;
+    --background-main-color: beige;
+}
+
+#main-nav {
+    display: flex;
+    justify-content: center;
+    border-bottom: 5px solid var(--border-nav-color);
+    background-color: var(--background-nav-color);
+}
+
+#landscapes {
+    display: flex;
+    justify-content: center;
+    gap: 1rem;
+    margin: 5rem 2rem 4rem 2rem;
+}
+
+.tile {
+    height: 700px;
+    max-width: 100px;
+    flex: 1;
+    border-radius: 30px;
+    transition: all 0.5s ease;
+    font-weight: bold;
+    font-size: 20px;
+    color: white;
+    padding: 15px;
+    display: flex;
+    align-items: flex-end;
+}
+
+.tile:hover {
+    transform: translateY(-15px);
+}
+
+[active^="true"] {
+    flex: 5;
+    max-width: 800px;
+}
+
+.first {
+    background-image: url("hq_1.jpg");
+    background-size: cover;
+    background-position: center;
+}
+
+.sec {
+    background-image: url("hq_2.jpg");
+    background-size: cover;
+    background-position: center;
```

Comment:
moze lepiej dziedziczyc czesc styli ? a nie powtarzac dwie linijki co element? 
Lub narzucic dwie klasy na element skoro to i tak sie powtarza, bo tak to mamy powtorzenia kodu czego musimy unikac